api/src/api/v1/community_businesses/volunteer_logs.ts

```diff
@@ -316,11 +316,17 @@ const routes: Hapi.ServerRoute[] = [
       const {
         server: { app: { knex } },
         pre: { communityBusiness },
-        payload,
+        payload: _payload,
       } = request;
 
       const { user, scope } = StandardCredentials.fromRequest(request);
 
+      /*
+       * HACK - Temp fix to be removed
+       * Silently ignore logs with invalid start dates
+       */
+      const payload = _payload.filter((log) => !isNaN(new Date(log.startedAt).valueOf()));
```

Comment:
@eliascodes can we add a condition so this hack only works for the organisation in question